test/test_proxy_tensor.py

```diff
@@ -443,6 +443,17 @@ def f():
         g = make_fx(f, tracing_mode=self.tracing_mode)()
         self.assertEqual(g(), f())
 
+    def test_constant_blowup(self):
+        def f():
+            val = torch.tensor([2])
+            blowup = val.repeat(1000)
```

Comment:
maybe instead, `val = torch.tensor([2]).repeat(1000); val.equal(val)` to test that we don't materialize after the `repeat(1000)` ? clear to me in this test we're not materializing `sum`, not clear to me that we're not materializing `blowup`